dotnet/src/Agents/Abstractions/KernelAgent.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.SemanticKernel.Agents;
+
+/// <summary>
+/// Base class for agents utilizing <see cref="Microsoft.SemanticKernel.Kernel"/> plugins or services.
+/// </summary>
+public abstract class KernelAgent : Agent
```

Comment:
Have you considered using an interface instead?

There is no behaviour provided here so if we had an interface it would provide more flexibility when designing class hierarchies.